### PR TITLE
Fixed avatar noPhoto background issues

### PR DESCRIPTION
**Pull request by mperrotti**
#### Related issues
Fixes https://meetup.atlassian.net/browse/WC-60

#### Description
Made background of avatar noPhoto opaque so avatars can overlap

#### Screenshots (if applicable)
screen shot 2017-07-06 at 5 44 27 pm

### CODE REVIEW

File: assets/scss/components/_avatar.scss [hunk 1]
```diff
@@ -75,7 +96,7 @@ Class                     | Description
 .avatar {
 	@extend %imgFill;
 	position: relative;
-	background-color: $C_textHint;
+	background-color: $avatarBg; // $C_textHint opaque, #888 with white scrim
```
Comment (akdetrick) [inline assets/scss/components/_avatar.scss]: This comment doesn't seem to be correct - `$avatarBg` maps directly to `$C_textHint`

File: assets/scss/components/_avatar.scss [hunk 2]
```diff
@@ -122,8 +143,15 @@ Class                     | Description
 .avatar--noPhoto {
 	position: relative;
 	text-indent: initial;
-	opacity: 0.54; // match text--secondary alpha
+	background-color: darken($avatarBgAsHex, ((1 - $noPhotoIconOpacity - alpha($avatarBg))*100)*1%);
 	background-position-y: 4px !important;
+	&:before {
+		background-color: transparentize($C_contentBG, 1-$noPhotoIconOpacity);
```
Comment (akdetrick) [inline assets/scss/components/_avatar.scss]: I'd recommend using `$C_white` instead. The content BG is a separate concept and may be subject to change (and in this case, we always want a white scrim to lighten the avatar)

File: assets/scss/components/_avatar.scss (hunk 1 above)
Comment (mperrotti) [inline assets/scss/components/_avatar.scss]: Oops, meant to delete this. Outdated comment from when I was playing around

File: assets/scss/components/_avatar.scss (hunk 2 above)
Comment (mperrotti) [inline assets/scss/components/_avatar.scss]: Makes sense

**Comment by akdetrick** [comment]
The `rgba-to-hex` color function PR in sasstools was just approved. Let's update sasstools in this branch to get that function and remove the definition here before we merge this

**Pull request merged**